aasemble/django/tests.py

```diff
@@ -107,3 +108,50 @@ def test_run_cmd_alternate_stdout(self):
 
         finally:
             os.unlink(tmpfile)
+
+
+def create_default_group(name):
+    # creating the group with all permissions
+    permissions = Permission.objects.all()
+    Group.objects.create(name=name)
+    group = Group.objects.get(name=name)
+    # We need to add each permission one by one
+    for permission in permissions:
+        group.permissions.add(permission)
+    return group
+
+
+def create_default_repo(name, username):
+    # Repo will be local repo.
+    # We will give a static key_id value to avoid any confict.
+    # geting the user. Make sure this already added before you pass it here
+    user = User.objects.get(username=username)
+    Repository.objects.create(user=user, name=name, key_id='12345')
+    return Repository.objects.get(name=name)
+
+
+def create_series(name, reponame):
+    # geting the repo. Make sure this already added before you pass it here
+    repo = Repository.objects.get(name=reponame)
+    Series.objects.create(name=name, repository=repo)
+    return Series.objects.get(name=name)
+
+
+def delete_repo(name):
+    repo = Repository.objects.get(name=name)
+    repo.delete()
+
+
+def delete_series(name):
+    series = Series.objects.get(name=name)
+    series.delete()
+
+
+def delete_group(name):
+    group = Group.objects.get(name=name)
+    group.delete()
+
+
+def delete_user(username):
+    user = User.objects.get(username=username)
+    user.delete()
```

Comment:
All of these methods are unnecessary. The fixtures provide groups, repos and series and the test infrastructure will perform the necessary cleanup.